From: Gunnar Wolf <gwolf@debian.org>
Origin: vendor
Forwarded: not-needed
Last-Update: 2009-12-03
Description: Set full path (instead of relative) to configuration migrators
 Migrator scripts which include the hardcoded relative path
 '../admin/CTK/CTK' are modified to include the correct full path

Index: cherokee/contrib/06to07.py
===================================================================
--- cherokee.orig/contrib/06to07.py	2008-12-26 20:31:59.000000000 -0600
+++ cherokee/contrib/06to07.py	2009-04-17 11:34:55.000000000 -0500
@@ -9,7 +9,7 @@
 
 import sys
 
-sys.path.append('../admin/CTK/CTK')
+sys.path.append('/usr/share/cherokee/admin/CTK/CTK')
 from Config import *
 
 def reparent_child (orig, target):
Index: cherokee/contrib/07to08.py
===================================================================
--- cherokee.orig/contrib/07to08.py	2008-12-26 20:31:59.000000000 -0600
+++ cherokee/contrib/07to08.py	2009-04-17 11:34:55.000000000 -0500
@@ -9,7 +9,7 @@
 
 import sys
 
-sys.path.append('../admin/CTK/CTK')
+sys.path.append('/usr/share/cherokee/admin/CTK/CTK')
 from Config import *
 
 
Index: cherokee/contrib/08to09.py
===================================================================
--- cherokee.orig/contrib/08to09.py	2008-12-26 20:31:59.000000000 -0600
+++ cherokee/contrib/08to09.py	2009-04-17 11:34:55.000000000 -0500
@@ -9,7 +9,7 @@
 
 import sys
 
-sys.path.append('../admin/CTK/CTK')
+sys.path.append('/usr/share/cherokee/admin/CTK/CTK')
 from Config import *
 
 def save_result (content, file):
Index: cherokee/contrib/0999to09910.py
===================================================================
--- cherokee.orig/contrib/0999to09910.py	2009-04-17 11:35:09.000000000 -0500
+++ cherokee/contrib/0999to09910.py	2009-04-17 11:35:30.000000000 -0500
@@ -9,7 +9,7 @@
 
 import sys
 
-sys.path.append('../admin/CTK/CTK')
+sys.path.append('/usr/share/cherokee/admin/CTK/CTK')
 from Config import *
 
 def save_result (content, file):
